napari/components/viewer_model.py

```diff
@@ -397,8 +413,6 @@ def _on_add_layer(self, event):
         layer = event.value
 
         # Connect individual layer events to viewer events
-        layer.events.select.connect(self._update_active_layer)
```

Comment:
hopefully with time we can move to having none of the event connections here and putting them all on the layer list